libs/wingsdk/src/target-sim/redis.inflight.ts

```diff
@@ -96,4 +90,42 @@ export class Redis implements IRedisClient, ISimulatorResourceInstance {
       throw new Error("Redis server not initialized");
     }
   }
+
+  public async set(key: string, value: string): Promise<void> {
+    let redis = await this.rawClient();
+    await redis.set(key, value);
+  }
+
+  public async get(key: string): Promise<string> {
+    let redis = await this.rawClient();
+    let result = await redis.get(key);
+    return result ?? "";
+  }
+
+  public async hset(key: string, field: string, value: string): Promise<void> {
```

Comment:
note that the (perhaps unexpected) return type here needs to be a wing `num` representing the number of new fields added to the hash as a result of this command.